libraries/lib-audio-io/AudioIO.cpp

```diff
@@ -573,6 +573,8 @@ bool AudioIO::StartPortAudioStream(const AudioIOStartStreamOptions &options,
          const PaHostApiInfo* hostInfo = Pa_GetHostApiInfo(playbackDeviceInfo->hostApi);
          bool isWASAPI = (hostInfo && hostInfo->type == paWASAPI);
          playbackParameters.suggestedLatency = isWASAPI ? 0.0 : latencyDuration/1000.0;
+
+         mUsingAlsa = (hostInfo && hostInfo->type == paALSA);
```

Comment:
This assignment happens only when `mSoftwarePlaythrough` is false.  I think it should be independent of that.

Reply:
Hmm, great catch!

Reply:
It happens too only when playing, not when recording only (without overdub).

Is there any similar issue with mixed up timing at the start of recording with ALSA?  But I am aware of no complaint.